b-lawlor/koha-plugin-batch-permissions-modifier
# Make batch-permissions-modifier an admin plugin

Since bug 36206 plugins can be configured to run as an admin only tool.

This patch changes the plugin method from 'tool' to 'admin' so that it is no longer exposed under the Tools module and must be run from the Administration module.

This would allow for administrators to give library staff permissions to run other less sensitive tool plugins like Label Maker without providing access to batch modify permissions.

To test:
- Apply patch and install plugin
- Login as superlibrarian                              
- Notice the plugin is not listed under Tools anymore
- Notice the plugin is listed under Admin > Plugins
